node/routes.py

```diff
@@ -97,10 +95,10 @@ def post_dispute() -> Response:
             "signature": utils.eth_sign(f'{zconfig.SEQUENCER["id"]}{timestamp}'),
         }
         return success_response(data=data)
-
-    zdb.init_txs(req_data["app_name"], req_data["txs"])
+    if req_data['app_name'] != '':
+        zdb.init_txs(req_data["app_name"], req_data["txs"])
     return error_response(ErrorCodes.ISSUE_NOT_FOUND)
-
+    
```

Comment:
is this extra whitespace?

Reply:
Fixed in this [commit](https://github.com/zellular-xyz/zsequencer/commit/ee5f0e7ebd51bb68f3c8d9b84186f11d1a58df64)